tap_facebook/__init__.py

```diff
@@ -210,12 +210,30 @@ class AdCreative(Stream):
 
     def __iter__(self):
         @retry_pattern(backoff.expo, FacebookRequestError, max_tries=5, factor=5)
-        def do_request():
+        def do_request(params):
             return self.account.get_ad_creatives(fields=self.fields(), # pylint: disable=no-member
-                                                 params={'limit': RESULT_RETURN_LIMIT})
-        ad_creative = do_request()
-        for a in ad_creative: # pylint: disable=invalid-name
-            yield {'record': a.export_all_data()}
+                                                 params={'limit': RESULT_RETURN_LIMIT, 'time_range': params})
+
+        # `time_range` wants a value in the shape of {"since":
+        # "YYYY-MM-DD", "until": "YYYY-MM-DD"}, which is what
+        # pendulum.parse().date() returns
+        date_window_start = pendulum.parse(CONFIG['start_date']).date()
```

Comment:
It appears that the previous behavior would always sync all of these (ie it did not use a bookmark). Is that right?

Reply:
Correct, and it should continue to sync all of them